src/datasets/tasks/text_classification.py

```diff
@@ -0,0 +1,40 @@
+from dataclasses import dataclass
+from typing import Dict, List
+
+from ..features import ClassLabel, Features, Value
+from .base import TaskTemplate
+
+
+@dataclass(frozen=True)
+class TextClassification(TaskTemplate):
+    task = "text-classification"
+    input_schema = Features({"text": Value("string")})
+    label_schema = Features({"labels": ClassLabel})
```

Comment:
Nit: Since we update this in `__post_init__` do we need to declare a default?

Reply:
good catch! i'll fix it :)

Reply:
ah actually we need to declare a default because `label_schema` is a required argument for `__init__`. i've reverted the change and added a TODO so we can think about a more elegant approach as we iterate on the other tasks